docs/usage/configuration-options.md

```diff
@@ -594,13 +594,40 @@ Check out our [Node.js documentation](https://docs.renovatebot.com/node) for a c
 
 ## npm
 
+The following dep types are currently supported by the npm manager :
+
+### dependencies
```

Comment:
No need to list these as subheadings. Just use a regular paragraph or list instead.